layouts/partials/posts_navigation.html

```diff
@@ -1,15 +1,23 @@
 {{ $skipPageNav := .Scratch.Get "skipPageNav" }}
 {{ if not (or .Params.skip_pagenav $skipPageNav) }}
-    {{ if or .NextInSection .PrevInSection }}
+    {{$prev := .PrevInSection}}
+    {{$next := .NextInSection}}
+    {{ with .Params.prevInSection }}
```

Comment:
This would have us use the parameter `prevInSection` in the front matter, but we just want `prev`.)